libs/editor/build.gradle

```diff
@@ -74,10 +67,31 @@ dependencies {
             strictly wordPressAztecVersion
         }
     }
+    
+    // Forcing version due to https://github.com/facebook/SoLoader/issues/94
+    // To be removed with React Native 0.70+
+    implementation("com.facebook.soloader:soloader") {
+        version {
+            strictly '0.10.4'
+        }
+    }
+
+    implementation 'com.facebook.fresco:fresco:2.5.0'
+    implementation 'com.facebook.fresco:imagepipeline-okhttp3:2.5.0'
+    implementation 'com.facebook.fbjni:fbjni:0.2.2'
+    implementation 'com.facebook.react:hermes-engine:0.69.4:release@aar'
+    implementation 'com.facebook.react:react-native:0.69.4:release@aar'
```

Comment:
> It also adds several implementations in libs/editor/build.gradle since they're not included with the React Native and Hermes binaries.

Are these the libraries not included in the React Native and Hermes binaries? I'm wondering if we'd need to define them here or if they could be part of the libraries required by Gutenberg Mobile, could we have them as [dependencies in the `react-native-bridge` build gradle file](https://github.com/WordPress/gutenberg/blob/3b04ce46f18d61aa25f0483ba49116e701835fdd/packages/react-native-bridge/android/react-native-bridge/build.gradle#L59)?

I noticed that we are also setting the React Native version, is this required? Would be a way to let the Gutenberg Mobile library to define this?

Reply:
Since we encountered issues with using the `debug` artifacts for both Hermes and React Native instead of the `release` ones, when excluding them and including them again some libraries needed to be referenced again.

I'm working on a draft PR as a follow-up to improve these, although to be able to force the release artifacts for all builds like (Jalapeno Debug) which are the builds PRs generate, we will have to keep a few implementations. 😅